compiler/noirc_frontend/src/utils.rs

```diff
@@ -0,0 +1,17 @@
+use acvm::{AcirField, FieldElement};
```

Comment:
I'm never too sure about introducing an `utils` module. It's probably better to put functions next to their types, but I wasn't sure if adding them to where `FieldElement` is was okay. There's also a function that operates on `BigInt` and we don't own that one. That said, I also see some other `utils.rs` files in the repo, so maybe it's fine :-)